kythe/go/extractors/config/base/testdata/expected_mvn_config.Dockerfile

```diff
@@ -14,11 +14,12 @@ ENV JAVA_HOME=/docker-java-home
 ENV PATH=$JAVA_HOME/bin:$PATH
 COPY --from=javac-extractor-artifacts /opt/kythe/extractors/runextractor /opt/kythe/extractors/runextractor
 COPY --from=javac-extractor-artifacts /opt/kythe/extractors/javac-wrapper.sh /opt/kythe/extractors/javac-wrapper.sh
-COPY --from=javac-extractor-artifacts /opt/kythe/extractors/javac_extractor.jar /opt/kythe/extractors/javac_extractor.jar
+COPY --from=javac-extractor-artifacts /opt/kythe/extractors/javac8_extractor.jar /opt/kythe/extractors/javac8_extractor.jar
```

Comment:
This should be in addition to `javac_extractor`.

Reply:
done